README.md

```diff
@@ -197,3 +237,11 @@ If `kubectl` is configured to configured with access to production, please take
 - `kind-cluster` - create a local [`kind`](https://github.com/Kubernetes-sigs/kind) cluster
 - `kind-load` - build and load container image for use in a `kind` cluster
 - `kind-rollout` - reload container image & config, then restart pods
+
+## Security
+
+See [CONTRIBUTING](CONTRIBUTING.md#security-issue-notifications) for more information.
+
+## License
```

Comment:
Should  you include a section that explains what is required to build the container image? Since in #63, you will be using a tool that is only available if the users installed docker from a specific source. For example, in AL2, using docker v20, I'll get this error

```sh
[ec2-user@ ~]$ cat /etc/os-release | grep NAME
NAME="Amazon Linux"
PRETTY_NAME="Amazon Linux 2"
CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2"
[ec2-user@ ~]$ docker --version
Docker version 20.10.4, build d3cb89e
[ec2-user@ ~]$ docker buildx
docker: 'buildx' is not a docker command.
See 'docker --help'
[ec2-user@~]$
```

Reply:
That's a good point. However, I am wondering if we are asking customers to build the `arm` image or we are going to publish ARM image as well. If we are going to publish it, I don't think we need to include a section related to building image.  If we are not, then we should definitely add the detailed information that you suggest. However, this information should be added as part of [PR-63](https://github.com/bottlerocket-os/bottlerocket-update-operator/pull/63) I think.

Reply:
I personally aslo feel we should get rid of **How to Contribute and Develop Changes** section from the readme. It adds noise, not everyone is going to develop. We can have a separate markdown file to add development details for anyone who is looking to help and link it in `CONTRIBUTING.md`. What do you think?

Reply:
I think we should be consistent across the projects. In the main OS repo, we have a document that explains how to build OS and the dependencies required. I understand that we will build the images, but users will try to build it as well for their own  purposes. Maybe they have an internal policy that doesn't allow them to use public images, and they have to setup their own image repository with the updater in it.

Reply:
Makes sense, I can create a separate issue to track this and address in another PR. Is that okay?

Reply:
Makes sense :+1:

Reply:
> #66 

Done